.github/actions/spell-check/whitelist.txt

```diff
@@ -0,0 +1,3240 @@
+OLECHAR
+onecore
+ONECOREBASE
+ONECOREREDIST
+ONECORESDKTOOLS
+ONECORESHELL
+onecoreuap
+onecoreuapuuid
+onecoreuuid
+ONECOREWINDOWS
+onehalf
+ONLCR
+Oo
+ools
+openbash
+opencode
+opencon
+openconsole
+opendb
+OPENIF
+OPENLINK
+openlogo
+openps
+opensdf
+opensource
+openvt
+openxmlformats
+oq
+ORIGINALFILENAME
+osc
+OSCBG
+OSCCT
+OSCFG
+OSCRCC
+OSCSCC
+OSCWT
+OSDEPENDSROOT
+osfhandle
+OSG
+OSGENG
+osgvsowi
+osgwiki
+osign
+ostream
+ostringstream
+Ou
+ouicompat
+outdir
+outfile
+Outof
+OUTOFCONTEXT
+OUTOFMEMORY
+outout
+OUTPATHROOT
+Outptr
+Ov
+OVERLAPPEDWINDOW
+OWNDC
+OWNERDRAWFIXED
+Owr
+packageuwp
+PACKCOORD
+PACKVERSION
+pagedown
+pageup
+PAINTPARAMS
+PAINTSTRUCT
+paket
+Paketen
+PALPC
+pankaj
+params
+parentable
+parms
+pasring
+passthrough
+PATCOPY
+pathcch
+PATTERNID
+paulcam
+pauldotknopf
+pb
+PBOOL
+pbxproj
+PBYTE
+pcat
+pcb
+pcch
+PCCHAR
+PCCONSOLE
+PCD
+pch
+PCHAR
+PCIDLIST
+PCIS
+PCLIENT
+PCLONG
+PCOBJECT
+pcon
+PCONSOLE
+PCONSOLEENDTASK
+PCONSOLESETFOREGROUND
+PCONSOLEWINDOWOWNER
+pcoord
+pcs
+pcshell
+PCSHORT
+PCSR
+PCSTR
+PCUNICODE
+PCWCH
+PCWCHAR
+PCWSTR
+pda
+pdb
+pdbonly
+pdbstr
+pdf
+pdp
+pdtobj
+pdw
+PDWORD
+pdx
+peb
+PEMAGIC
+PENDTASKMSG
+pfa
+PFACENODE
+pfed
+pfi
+PFILE
+pfn
+PFNCONSOLECREATEIOTHREAD
+PFONT
+PFONTENUMDATA
+PFS
+pgc
+pgd
+pgdn
+pgp
+pguid
+pgup
+Pham
+PHANDLE
+phhook
+phsl
+phwnd
+pid
+pidb
+pidl
+PIDLIST
+pii
+pinvoke
+pipename
+pipestr
+pixelheight
+PIXELSLIST
+Pj
+PJOBOBJECT
+pkey
+pkginfo
+placeholders
+platforming
+playsound
+PLEHMQNl
+plg
+plist
+PLOGICAL
+plugin
+PMv
+png
+pnm
+PNMLINK
+pntm
+PNTSTATUS
+POBJECT
+Podcast
+pointerpoint
+POINTSLIST
+polytextoutw
+POLYTEXTW
+popclip
+popd
+POPF
+poppack
+popup
+POPUPATTR
+PORFLG
+positionals
+POSTCHARBREAKS
+POSX
+POSXSCROLL
+POSYSCROLL
+powershell
+ppci
+ppf
+ppguid
+ppidl
+pplx
+PPORT
+PPROC
+PPROCESS
+ppropvar
+ppsi
+ppsl
+ppsp
+ppsz
+ppv
+ppwch
+PQRST
+pragma
+prc
+prealigned
+prebuilt
+precomp
+prect
+prefast
+prefilled
+prefs
+PRELOAD
+PREMULTIPLIED
+prepopulated
+PREVENTPINNING
+PREVIEWLABEL
+PREVIEWWINDOW
+PREVLINE
+prg
+pri
+printf
+prioritised
+prioritization
+processenv
+processhost
+PROCESSINFOCLASS
+proj
+Projekt
+PROPERTYID
+PROPERTYKEY
+PROPERTYVAL
+propsheet
+PROPSHEETHEADER
+PROPSHEETPAGE
+propslib
+propsys
+PROPTITLE
+propvar
+propvariant
+propvarutil
+psa
+psd
+PSECURITY
+psess
+pseudocode
+pseudoconsole
+pseudoterminal
+psh
+pshn
+PSHNOTIFY
+PSHORT
+pshpack
+PSINGLE
+psl
+psldl
+psm
+PSMALL
+PSNRET
+PSobject
+psp
+PSPCB
+psr
+PSTR
+psuedoconsole
+psz
+ptch
+ptr
+ptrdiff
+ptsz
+PTYIn
+publishproj
+publishsettings
+pubxml
+PUCHAR
+PULONG
+PUNICODE
+pushd
+putchar
+putwchar
+pv
+PVOID
+pw
+pwch
+PWCHAR
+PWDDMCONSOLECONTEXT
+PWORD
+pwsh
+pwstr
+pwsz
+px
+py
+pyc
+pythonw
+qi
+QJ
+qo
+QQQQQ
+QQQQQQQQQ
+QQQQQQQQQQ
+QQQQQQQQQQABCDEFGHIJ
+QQQQQQQQQQABCDEFGHIJKLMNOPQRSTQQQQQQQQQ
+QQQQQQQQQQABCDEFGHIJKLMNOPQRSTQQQQQQQQQQ
+QQQQQQQQQQABCDEFGHIJPQRSTQQQQQQQQQQ
+qrstuvwxyz
+qsort
+queryable
+quickedit
+Qw
+QWER
+qwerty
+QWERTYUIOP
+qwertyuiopasdfg
```

Comment:
ok this has to be test code or something. maybe we need a test code exclusion or the quote exclusion (or both configurably?)

Reply:
Not actually test code, sort of documented sample input:
https://github.com/microsoft/terminal/blob/be77b2a2e98420db9d275d68f35b55ac58bc93c0/src/host/selectionInput.cpp#L414
https://github.com/microsoft/terminal/blob/be77b2a2e98420db9d275d68f35b55ac58bc93c0/src/host/selectionInput.cpp#L455

Reply:
Hm. Alright.

Reply:
Note: when new misspellings are added, the tool automatically annotates them. Which makes finding them trivial.

It's possible for the tool to print out samples for existing things, but I've worried about base64 or binary garbage, and for large output runs, it isn't a great idea to flood people.

My personal tooling from which this tool grew has `g` which I use to find instances...

I'm open to whether there should be a flag for samples (just one filename per token, or one line from one file). Would you want that? – I'd imagine it'd be overwhelming...